### ISSUE TITLE
Prisma 1.0: New CLI commands layout

### ISSUE THREAD

**Issue by schickling** [graphcool/prisma#1330]
```
Database service:
  init            Initialize a new service
  deploy          Deploy service changes (or new service)
  info            Display service information (endpoints, cluster, ...)
  token           Create a new service token
  list            List all deployed services
  delete          Delete an existing service

Data workflows:
  playground      Open service endpoints in GraphQL Playground
  import          Import data into a service
  export          Export service data to local file
  reset           Reset the stage data

Cloud:
  login           Login or signup to Graphcool Cloud
  console         Open Graphcool Console in browser
  account         Display account information

Local development:
  local start     Start local development cluster
  local stop      Stop local development cluster
  local upgrade   Upgrade to the latest (or specific) cluster version
  local nuke      Hard-reset local development cluster

Clusters:
  cluster list    List all clusters
  cluster logs    Output cluster logs
  cluster add     Add an existing cluster
  cluster remove  Remove cluster
```

**Comment by sorenbs** [comment]
@timsuchanek please add the `proposal/accepted` label when you think this is final

**Comment by sorenbs** [comment]
the `graphcool playground` command should generate a jwt that is valid for the service and automatically add the header:

```
Authorization=Bearer ${valid jwt}
```

As described in [1328](https://github.com/graphcool/framework/issues/1328#issuecomment-349711517) each stage has their own secret, so the CLI or Playground has to support this somehow.

**Comment by marktani** [comment]
`list            List all deployed services`
`clusters        List all clusters`

* I prefer `services` and `clusters`.
* Also, I propose a `--cluster` parameter for `services` to only list services for a certain cluster:

```
graphcool services # services of all clusters
graphcool services -c local # only local services
```

**Comment by kbrandwijk** [comment]
@marktani Also related to https://github.com/graphcool/framework/issues/1100#issuecomment-341281136

**Comment by schickling** [comment]
The reasoning behind `list` instead of `services` was to make services the "default entity". That's why there's also `graphcool delete` and `graphcool deploy`. While I see your point of this seeming inconsistent, I think this is a solid tradeoff.

**Comment by marktani** [comment]
Makes sense, @schickling!

I still think `--clusters` would help a lot, I am usually interested in services for a specific cluster and all other services are just noise.

**Comment by marktani** [comment]
@timsuchanek also keep https://github.com/graphcool/framework/issues/1408 in mind 🙂

**Comment by marktani** [comment]
`graphcool clusters` should show cluster version

**Comment by sorenbs** [comment]
Would it make sense to turn 

`gc clusters`

into one of:

`gc cluster ls`
`gc cluster list`

This would enable us to add more cluster commands in the future. It also fit with commands such as `docker volume ls`

**Comment by schickling** [comment]
Thanks a lot for this feedback @sorenbs. Here is a new design proposal:

![image](https://user-images.githubusercontent.com/1567498/34462511-d8d16f3c-ee45-11e7-9798-5aa42a8730bb.png)

**Comment by kbrandwijk** [comment]
How will `gc list` function with regards to clusters (https://github.com/graphcool/framework/issues/1330#issuecomment-352189293)?

**Comment by schickling** [comment]
@kbrandwijk 
![image](https://user-images.githubusercontent.com/1567498/34462536-3e90eb9a-ee46-11e7-8df8-22f0fc22ba13.png)

**Comment by marktani** [comment]
@kbrandwijk I opened a separate discussion about `gc list` here: https://github.com/graphcool/framework/issues/1493

Currently, all stages of all services are listed. With my proposed behaviour in the linked issue, I don't think a `--cluster` switch is necessary anymore.

**Comment by marktani** [comment]
This has been implemented in [Prisma 1.0](https://github.com/graphcool/prisma/releases/tag/1.0.0). For a reference list of CLI commands, [consult the documentation](https://www.prismagraphql.com/docs/reference/).
